scaling_performance/host_practices.adoc

```diff
@@ -137,32 +137,39 @@ etcd processes are typically memory intensive. Master / API server processes are
 ====
 
 [[scaling-performance-capacity-tuned-profile]]
-== Scaling Hosts Using the TuneD Profile
+== Scaling Hosts Using the Tuned Profile
 
-TuneD is a tuning profile delivery mechanism enabled by default in Red Hat
-Enterprise Linux and other Red Hat products. TuneD customizes Linux settings,
+**Tuned** is a tuning profile delivery mechanism enabled by default in Red Hat
```

Comment:
@jmencak In the RHEL docs, Tuned is consistently referred to as the **tuned-adm** tool. Should that be the case across this topic, or is calling it **Tuned** most appropriate?

Reply:
tuned-adm tool is just the Tuned client.  Imagine it as a subset of Tuned.  Tuned is the whole package/system of setting profiles.

Reply:
@jmencak Got it. Thanks for clarifying!